src/components/todoitems/TodoItems.js

```diff
@@ -4,28 +4,28 @@ import { FaTrash } from 'react-icons/fa';
 
 function TodoItems({ onCheck, onClick, data }) {
 
-    const labelRef = useRef()
+    const labelRef = useRef();
 
     useEffect(() => {
-        const label = labelRef.current
-        label.className = data.completed ? "todoTextComplete" : "todoText"
-    }, [data.completed])
+        const label = labelRef.current;
+        label.className = data.completed ? "todoTextComplete" : "todoText";
+    }, [data.completed]);
 
     const checkHandler = () => {
-        onCheck(data.id)
-    }
+        onCheck(data.id);
+    };
 
     const clickHandler = () => {
-        onClick(data.id)
-    }
+        onClick(data.id);
+    };
 
     return <div className='todoitem' >
         <div className='todoItemWrapper'>
             <input className='checkBox' type='checkbox' onChange={checkHandler} checked={data.completed} />
-            <label ref={labelRef}className='todoText'>{data.text}</label>
+            <label ref={labelRef} className='todoText'>{data.todo}</label>
```

Comment:
use template string here to add classname for  complete/incomplete scenario

Reply:
done